sycl/include/CL/sycl/detail/cg.hpp

```diff
@@ -427,10 +445,11 @@ class CGFill : public CG {
          vector_class<detail::AccessorImplPtr> AccStorage,
          vector_class<shared_ptr_class<const void>> SharedPtrStorage,
          vector_class<Requirement *> Requirements,
-         vector_class<detail::EventImplPtr> Events)
+         vector_class<detail::EventImplPtr> Events,
+          cl::sycl::detail::code_location loc = {})
```

Comment:
Can we replace `cl::sycl::detail::` with `detail::` in this file for all the new instances to keep the style consistent, please?
All other parameters omit `cl::sycl` type qualification.

Reply:
Done!